plugins/github-issues/package.json

```diff
@@ -26,15 +26,17 @@
     "@backstage/catalog-model": "^1.0.3",
     "@backstage/core-components": "^0.10.1-next.0",
     "@backstage/core-plugin-api": "^1.0.5-next.0",
+    "@backstage/errors": "^1.1.0",
     "@backstage/integration": "^1.3.0-next.0",
     "@backstage/plugin-catalog-react": "^1.1.3-next.0",
     "@backstage/theme": "^0.2.15",
+    "@faker-js/faker": "^7.4.0",
```

Comment:
This should move into the devDependencies section.

Reply:
Thanks, implemented here: [c668c85](https://github.com/backstage/backstage/pull/13059/commits/c668c850a4bc177f8574c7a452d346986c56501a)